bowling.test.js

```diff
@@ -0,0 +1,48 @@
+const Bowling = require('./bowling.js');
+
+const rollMultiple = (pins, rolls) => {
+  for (let i = 0; i < rolls; i++) {
+    game.roll(pins);
+  }
+} 
```

Comment:
Awesome method. And, considering that your bowling scorecard is mainly for calculating the final score, I think you could move this method to your Bowling class! This way, you could utilise it, not only in your test, but when running the code itself.